src/index.ts

```diff
@@ -179,7 +193,15 @@ export async function compile(schema: JSONSchema4, name: string, options: Partia
   const formatted = format(generated, _options)
   log('white', 'formatter', time(), '✅ Result:', formatted)
 
-  return formatted
+  return {
+    typescript: formatted,
```

Comment:
This is a breaking change. Please avoid changing the result shape, and instead focus on making the most targeted change possible.